### PR TITLE
Optimize images with oxipng "max compression"

### PR DESCRIPTION
**Pull request by ghost**
Command used to perform compression:
`oxipng -o max --strip all --zopfli --zi 255 --recursive .`

NOTE: `--zi` flag appers only in `oxipng` of version 9.1.3 which was released only yesterday

All the images got benefit though it is small for some of them it is 3 bytes It optimized all the `**/icon_background.png` better. Now they use 1 bit encoding instead of 8 bit. Files from `dist/**/*ios*/*.png` will have benefit of 3 bytes and weren't included (even though this commit include one or two other files with size diff 3 byte). IOS images are very long to compute....

### CODE REVIEW

**Comment by dustdfg** [comment]
Why? I've ran it just for fun because why not to try the best compression ever?

If all the images would have difference of 3bytes there wouldn't be sense in it but all the `**/icon_background.png` got benefit. It deletes unnecessary metadata from `icon_monochrome.png` and compresses `logo_outlined`...

**Comment by dustdfg** [comment]
As maintainers you should check for any malware and etc... And run it by yourself though I am not sure this program is very reproducible due to one bug/issue on github of the project... Anyway if you run it just be aware. Be ready to go for a very long walk even if your machine compiles godot in 10minutes... It is extremely slow... It doesn't worth spending so much time for it... But I did just for fun... Actually twice... I accidentally wiped results of first attempt so it is a reason why second attempt doesn't include ios image :)

**Comment by dustdfg** [comment]
And just a funny fact. I understand that godot is used by people working on different platforms. One of them doesn't respect normal slashes and symbolic links. So repo doesn't use symlinks for identical (in byte sense) images... But IOS images are strange... They identical and are placed in the same dir... 
screen-1732943662
But names suggest they should differ...

File: icon.png
Comment (aaronfranke) [inline icon.png:1]: Considering that this PR increases the Git history size, I don't think we should accept any changes to PNG files that result in such an incredibly small difference.

I'd only recommend keeping changes when the difference in size was significant, not just 1% or 2% or even 0% as is the case for this image.

**Comment by dustdfg** [comment]
Superseded by https://github.com/godotengine/godot/pull/99908

**Pull request closed without merging**